src/webgl.js

```diff
@@ -188,7 +183,7 @@ function initCopyProgram(gl) {
       copyOut = texture(originalSampler, vec2((gl_FragCoord.x - renderArea.x) / videoRes.x, 1.0 - ((gl_FragCoord.y - renderArea.y) / videoRes.y)));
     }
   }
-  `;
+`;
```

Comment:
nit... shouldn't this still be intended since it's defined in an indented block?

Reply:
Fixed!